ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -209,6 +209,24 @@ def allclose(self, other, rtol=1e-05, atol=1e-08, equal_nan=False, name=None):
     def sort(self, axis=-1, descending=False, name=None):
         return ivy.sort(self._ivy_array, axis=axis, descending=descending)
 
+    @with_supported_dtypes(
+        {
+            "2.4.2 and below": (
+                "bool",
+                "int8",
+                "int16",
+                "int32",
+                "int64",
+                "float32",
+                "float64",
+            )
+        },
+        "paddle",
+    )
+    def logical_or(self, y, out=None, name=None):
+        y_ivy = _to_ivy_array(y)

```

Comment:
You need to wrap the variables explicitly for frontend functions.

Reply:
could please you explain why do I need  wrap the variables explicitly for frontend functions?
 _to_ivy_array(y) convert it into ivy_array

Reply:
@Dsantra92  are you talking about this way to implemet the function ?

```
def logical_or(self, y, out=None, name=None):
        return paddle_frontend.logical_or(self, y, out=out)
```

Reply:
> @Dsantra92 are you talking about this way to implemet the function ?
> 
> ```
> def logical_or(self, y, out=None, name=None):
>         return paddle_frontend.logical_or(self, y, out=out)
> ```
Yeah, this is what @Dsantra92 meant probably.